tool/instrument/instrument.go

```diff
@@ -145,6 +145,19 @@ func matchCompileImportPath(importPath string, args []string) bool {
 	return false
 }
 
+func stripComplete(args []string) []string {
+	// -complete flag is used to indicate compiling complete package
+	// (no C or assembly), since we generate may local function hook names
+	// in trampoline func,
```

Comment:
`-complete` is passed when there are non-go files like .cgo. We don't create any of these kinds of files, why we need to remove this flag?

Reply:
Because compiler complains about
```
$WORK/b005/otel_inst_file_p4.go:190:6: missing function body
$WORK/b005/otel_inst_file_p4.go:462:6: missing function body
```

Reply:
Improve the doc:
By using `go:linkname` mechanism to bind the implementation of the hooks to the injection points, it is necessary to remove `-complete` option from compilation arguments.